guide/src/class.md

```diff
@@ -52,18 +52,23 @@ enum HttpResponse {
     // ...
 }
 
-// PyO3 also supports enums with non-unit variants
-// These complex enums have sligtly different behavior from the simple enums above
-// They are meant to work with instance checks and match statement patterns
+/// PyO3 also supports enums with Struct and Tuple variants
```

Comment:
Why change these to a doc comment, it's not like this would be a proper doc comment for the `enum Shape`.